node/resolvers/profile/services.ts

```diff
@@ -44,11 +44,13 @@ export function getPasswordLastUpdate(context: Context) {
 
   if (!userCookie) return null
 
-  return makeRequest(context.vtex, url, 'GET', undefined, userCookie).then(
-    (response: any) => {
-      return response.data.passwordLastUpdate
-    }
-  )
+  return makeRequest({
+    ctx: context.vtex,
+    url,
+    authCookieAdmin: userCookie,
```

Comment:
Shouldn't it be authCookieStore?

Reply:
That's what I thought, but we found out that the API only reads the admin cookie in this endpoint.
Apparently, this is a hack to pass the store auth cookie as the admin one. I didn't want to touch this and possibly break something

![image](https://user-images.githubusercontent.com/22064061/102803385-ede07e00-4396-11eb-8f41-8b0bd6d81003.png)